src/cockpit/samples.py

```diff
@@ -39,6 +42,10 @@ def read_int_file(rootfd: int, statfile: str, default: Optional[int] = None, key
 
     try:
         data = os.read(fd, 1024)
+    except OSError as e:
+        # can run into ENODEV
+        logger.info('Failed to read %s: %s', statfile, e)
```

Comment:
Info, shouldn't this be error?

Reply:
We could do this in a more fine-grained manner -- e.g. logger.debug() for ENODEV or ENOENT (expected cases), and logger.warning() for others?

Reply:
ENOENT makes no sense for read(). I adjusted it to be silent for ENODEV, and warn for other errors.